azure-pipelines/README.md

```diff
@@ -28,30 +28,18 @@ schedules:
       include:
         - main
 
-# `resources` specifies the location of templates to pick up, use it to get 1ES and AzExt templates
+# `resources` specifies the location of templates to pick up, use it to get AzExt templates
 resources:
   repositories:
-    - repository: 1esPipelines
-      type: git
-      name: 1ESPipelineTemplates/1ESPipelineTemplates
-      ref: refs/tags/release
     - repository: azExtTemplates
       type: github
       name: microsoft/vscode-azuretools
-      ref: main
+      ref: bmw/1es4

```

Comment:
does this need to be changed back to main?

Reply:
Yes it does, good catch